com.valvesoftware.Steam.json

```diff
@@ -42,11 +43,6 @@
         "--env=ALSA_CONFIG_PATH=",
         "--env=MESA_GLSL_CACHE_DIR="
     ],
-    "build-options" : {
```

Comment:
Why was this removed

Reply:
Because it's only needed in the steam module: `"PREFIX=/app make install"`